modules/ocf/manifests/serial_getty.pp

```diff
@@ -0,0 +1,8 @@
+class ocf::serial_getty {
+  service { 'serial-getty@ttyS0':
+    ensure   => running,
```

Comment:
Can you enable with an exec and notify the service from the exec (or subscribe the other way)?